src/mixins/filing-template-mixin.ts

```diff
@@ -111,7 +113,14 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       filing.filing.incorporationApplication.nameRequest.legalName = this.getApprovedName
     }
 
-    // If this is a future effective filing then add the effective date.
+    // If this is a Cooperative then save the cooperative association type.
+    if (this.isTypeCoop) {
```

Comment:
Please move this into the switch statement above.

Reply:
Ahh will look into this.

Reply:
didn't see that case before! Fixed in next commit.